lib/tool_belt/cherry_picker.rb

```diff
@@ -92,7 +93,11 @@ def write_cherry_pick_log(picks, release)
     end
 
     def log_entry(pick)
-      "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      if pick['bugzilla_id']
+        "#{pick['bugzilla_id']} - #{pick['id']} : [#{pick['revision']}] #{pick['subject']}"
+      else
+        "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      end
```

Comment:
This code is not my final solution; this was a minimally invasive way to keep upstream functionality in place and add downstream features.